src/main/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBase.scala

```diff
@@ -102,12 +105,11 @@ trait DataFrameSuiteBaseLike extends SparkContextProvider
   }
 
   /**
-   * Compares if two [[DataFrame]]s are equal, checks the schema and then if that
-   * matches checks if the rows are equal.
-   */
+    * Compares if two [[DataFrame]]s are equal, checks the schema and then if that
```

Comment:
Did not notice this, IDE preferences seem to have screwed it up. Will fix them in them.